packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/Column.java

```diff
@@ -123,13 +131,19 @@ public Builder label(String label) {
             return this;
         }
 
+        public Builder header(String header) {
+            this.header = Objects.requireNonNull(header);
+            return this;
+        }
+
         public Column build() {
             Column column = new Column();
             column.id = Objects.requireNonNull(this.id);
             column.descriptionId = Objects.requireNonNull(this.descriptionId);
             column.targetObjectId = Objects.requireNonNull(this.targetObjectId);
             column.targetObjectKind = Objects.requireNonNull(this.targetObjectKind);
             column.label = Objects.requireNonNull(this.label);
+            column.header = this.header; //Optional on purpose
```

Comment:
No, don't make things optional, it creates tons of complexity for nothing. It forces us to embrace defensive programming and it would just create issues.